src/commands/deploy.ts

```diff
@@ -351,3 +351,28 @@ async function runPreDeployTask(deployFsPath: string, telemetryProperties: Telem
         }
     }
 }
+
+async function verifyWebContentSettings(node: FunctionAppTreeItem, telemetryProperties: TelemetryProperties): Promise<void> {
+    if (node.isLinuxPreview) {
+        // we need this check due to this issue: https://github.com/Microsoft/vscode-azurefunctions/issues/625
+        const client: SiteClient = node.root.client;
+        const applicationSettings: StringDictionary = await client.listApplicationSettings();
+        const WEBSITE_CONTENTAZUREFILECONNECTIONSTRING: string = 'WEBSITE_CONTENTAZUREFILECONNECTIONSTRING';
+        const WEBSITE_CONTENTSHARE: string = 'WEBSITE_CONTENTSHARE';
+        if (applicationSettings.properties && (applicationSettings.properties[WEBSITE_CONTENTAZUREFILECONNECTIONSTRING] || applicationSettings.properties[WEBSITE_CONTENTSHARE])) {
+            telemetryProperties.webContentSettingsRemoved = 'false ';
+            await ext.ui.showWarningMessage(
+                localize('notConfiguredForDeploy', 'The selected app is not configured for deployment through VS Code. Remove app settings "{0}" and "{1}"?', WEBSITE_CONTENTAZUREFILECONNECTIONSTRING, WEBSITE_CONTENTSHARE),
+                { modal: true },
+                DialogResponses.yes,
+                DialogResponses.cancel
+            );
+            delete applicationSettings.properties[WEBSITE_CONTENTAZUREFILECONNECTIONSTRING];
+            delete applicationSettings.properties[WEBSITE_CONTENTSHARE];
+            telemetryProperties.webContentSettingsRemoved = 'true';
+            await client.updateApplicationSettings(applicationSettings);
+            // if the user cancels the deployment, the app settings node doesn't reflect the deleted settings
+            await node.refresh();
```

Comment:
You can just call `node. _appSettingsTreeItem.refresh()` which would be faster since it doesn't have to refresh other stuff

(Of course you'd have to make that tree item public)

Reply:
I'm not super concerned about this-- it seems like something that won't come up very often.